From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @azdocprs, has been notified to review your proposed change.

tysonn commented:
@MikeWasson - Mike, could you review this proposed update, please?

MikeWasson commented on includes/guidance-compute-single-vm-windows.md:
```diff
@@ -82,7 +82,7 @@ Provisioning VM in Azure involves more moving parts than just the VM itself. The
 - Your VM may be affected by [planned maintenance][planned-maintenance] or [unplanned maintenance][manage-vm-availability]. You can use [VM reboot logs][reboot-logs] to determine whether a VM reboot was caused by planned maintenance.
 
-- VHDs are backed by [Azure Storage][azure-storage], which is replicated for durability and availability.
+- VHDs are backed up by [Azure Storage][azure-storage], which is replicated for durability and availability.
```
"backed up" would mean a copy, "backed" is actually correct here (Azure Storage is the backing storage for the VHD)

MikeWasson commented:
One comment. The rest look good, thanks!

4c74356b41 commented:
i think i reverted it

MikeWasson commented:
Looks good

4c74356b41 commented:
@MikeWasson thanks
@tysonn Yeah, I'm sorry to be such an pain in the neck, but could you perhaps look into other PR's i got (plenty of them, tbh), some of them are like 3 weeks old already?

MikeWasson commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

qinezh committed:
Remove extra backtick

qinezh committed:
Fix invalid bookmark

SnehaGunda committed:
Adding ASA overview topic

SnehaGunda committed:
updating header

SnehaGunda committed:
Updating intent statement

SnehaGunda committed:
Updating delivery mechanism

carolinacmoravia committed:
C15290: LOCBLOCKER: Missing ``` blocking text for localization 

Hello, @ckarst, 

Localization team has reported source content issue that causes localizable text to be blocked from localization.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
M26062: Possible error: "may be" should be "may not be"

Hello, @jimdial, 

Translator has reported possible source content issue.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

carolinacmoravia committed:
C15294: LOCBLOCKER: Unloc strings due to extra spaces 

Hello, @JYOTHIRMAISURI, 

Localization team has reported possible source content issue.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

BarbaraSelden committed:
saas images final

carolinacmoravia committed:
C15286: Missing ``` for code text

Hello, @jwhitedev, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

mimig1 committed:
Cosmos DB: Replicating usability fixes to all QS

mimig1 committed:
date updates

carolinacmoravia committed:
C15816: LOCBLOCKER: Extra character causing content blocked for localization

Hello, @YasinMSFT,

Localization team has reported source content issue that causes localizable text to be blocked from localization.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

mimig1 committed:
more updates

mimig1 committed:
pushed usability updates to cassandra, mongo, and table QS

SnehaGunda committed:
Merging with master

SnehaGunda committed:
Updating image

subsarma committed:
Added TXT ascii record support details in DNS FAQ

subsarma committed:
fixed API and SDK version

Commit:
freshness

Commit:
more fixing metadata

mimig1 committed:
fixing validation issues

mimig1 committed:
fixed acrolinx

mattchenderson committed:
removing slot warning, updating CLI

Commit:
updating PHP Versioning

littleaj committed:
fixed JRE support statements to show correct versions and remove implementation qualifiers

Commit:
update metadata

Commit:
updated composer instructions

Commit:
updating topic based on Acrolinx feedback

Commit:
updating topic based on Acrolinx feedback round 2

Commit:
corrected runbook name

v-nagta committed:
Missing icons update

v-nagta committed:
Merge branch 'master' into missingiconsv3

linda33wj committed:
Update new MongoDB support

Commit:
updates

mimig1 committed:
QA updates

Commit:
fixing merge conflicts

mimig1 committed:
Additional formatting fix

mimig1 committed:
Code indentation fix.

mimig1 committed:
Added missing step.

Commit:
Updates for service tags and default rules

Commit:
update

rwike77 committed:
updated authors

PRMerger17 committed:
Merge pull request #37366 from carolinacmoravia/patch-671

C15290: LOCBLOCKER: Missing ``` blocking text for localization

v-shils committed:
Corrected typo.

PRMerger18 committed:
Merge pull request #37759 from linda33wj/master

Update new MongoDB support

carolinacmoravia committed:
Update traffic-analytics-faq.md

Commit:
Update to bullet item

PRMerger19 committed:
Merge pull request #37392 from carolinacmoravia/patch-673

C15294: LOCBLOCKER: Unloc strings due to extra spaces

rwike77 committed:
acrolinx updates

v-shils committed:
Merge pull request #37749 from georgewallace/issue7137

automation: corrected runbook name

Commit:
Merge pull request #37784 from Minewiskan/aas0413

Update to bullet item

PRMerger10 committed:
Merge pull request #37501 from carolinacmoravia/patch-674

C15286: Missing ``` for code text

PRMerger10 committed:
Merge pull request #37563 from carolinacmoravia/patch-676

C15816: LOCBLOCKER: Extra character causing content blocked for local…

MGoedtel committed:
added specific versions supported

MGoedtel committed:
updated per Acrolinx

mattchenderson committed:
fixing statement around resource groups for linux web apps

MGoedtel committed:
updated file per other suggestions from Acrolinx

MGoedtel committed:
updated file per other suggestions from Acrolinx

PRMerger17 committed:
Merge pull request #37720 from mattchenderson/msiupdate

removing slot warning, updating CLI

v-shils committed:
Merge pull request #37764 from Blackmist/hacking

BULK METADATA FIXES

Commit:
Update analysis-services-gateway-install.md

MicrosoftGuyJFlo committed:
[Azure AD] WHFB - Take ownership to delete and redirect

Commit:
Fixed typo and small improvements

Commit:
Merge pull request #37788 from MGoedtel/UpdateNagiosAlert413

added specific versions supported

Commit:
Merge pull request #37795 from Minewiskan/patch-1

Update analysis-services-gateway-install.md

Commit:
update CLI Instructions for Azure CLI 2.0

MGoedtel committed:
updated article again

SnehaGunda committed:
Updating per CFR feedback

PRMerger12 committed:
Merge pull request #37380 from carolinacmoravia/patch-672

M26062: Possible error: "may be" should be "may not be"

SnehaGunda committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ASAOverview

v-shils committed:
Merge pull request #37780 from rwike77/authorchange

updated authors

PRMerger14 committed:
Merge pull request #37796 from marco-rossi29/patch-2

Fixed typo and small improvements

MGoedtel committed:
minor update made

pareshmu committed:
Update expressroutelocation page

PRMerger15 committed:
Merge pull request #37799 from MicrosoftGuyJFlo/WHFB-TakeOwnership

[Azure AD] WHFB - Take ownership to delete and redirect

mimig1 committed:
Cosmos DB: Corrected incorrect instances of CosmosDB.

mimig1 committed:
additional fix

PRMerger17 committed:
Merge pull request #37800 from MGoedtel/UpdateNagiosAlertB413

updated article again

MicrosoftGuyJFlo committed:
[Azure AD] WHFB - Redirect and delete

mimig1 committed:
added meta to include files

PRMerger18 committed:
Merge pull request #37803 from pareshmu/patch-4

Update expressroutelocation page

pareshmu committed:
Updated er locations - providers

rachel-msft committed:
update note for georestore

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into georup

Commit:
Merge pull request #37809 from pareshmu/patch-4

Updated er locations - providers

Commit:
Bookmark fix

GitHubber17 committed:
Merge pull request #37730 from SyntaxC4-MSFT/cfowler-php-updates

Addressing PHP Feddback

rmca14 committed:
Merge pull request #37758 from v-nagta/missingiconsv3

Missing icons update

Commit:
Merge pull request #37805 from mimig1/patch-237

Cosmos DB: Corrected incorrect instances of CosmosDB.

GitHubber17 committed:
Merge pull request #37790 from mattchenderson/linuxfix

fixing statement around resource groups for linux web apps

rolyon committed:
Incorporated feedback from CarnegieJ in PR #6588

GitHubber17 committed:
Merge pull request #37614 from mimig1/04102018usability

Cosmos DB: Replicating usability fixes to all QS

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-psroledefobject-pr6588

SnehaGunda committed:
Minor update

barbkess committed:
update metadata

amanarneja committed:
Fixing Links 

Fixes MicrosoftDocs/azure-docs#7131

amanarneja committed:
Removing some unwanted characters

rachel-msft committed:
tabs

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into georup

PRMerger17 committed:
Merge pull request #37771 from jimdial/patch-152

Updates for service tags and default rules

barbkess committed:
remove 2 migration topics

PRMerger18 committed:
Merge pull request #37724 from littleaj/appinsights_javasdk_jre_support

Update JRE support statments for AppInsights

rachel-msft committed:
remove extension section

barbkess committed:
redirect

amanarneja committed:
Updating to fix a warning

tynevi committed:
Merge pull request #29271 from qinezh/patch-2

Fix invalid bookmark

tynevi committed:
Merge pull request #29270 from qinezh/patch-1

Remove extra backtick

GitHubber17 committed:
Merge pull request #37808 from MicrosoftGuyJFlo/WHFB-RedirectAndDelete

[Azure AD] WHFB - Redirect and delete

barbkess committed:
fix merge conflict

SnehaGunda committed:
Acrolinx updates

GitHubber17 committed:
Merge pull request #37814 from barbkess/topic-updates2

update metadata

Commit:
Merge pull request #37664 from subsarma/patch-3

Added TXT extended ascii support details in DNS FAQ

SnehaGunda committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ASAOverview

PRMerger10 committed:
Merge pull request #37810 from rachel-msft/georup

Georup

amanarneja committed:
Fixed corrupted header

rachel-msft committed:
lost a space

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger15 committed:
Merge pull request #37816 from amanarneja/patch-27

Fixing Links

Ja-Dunn committed:
Merge pull request #37481 from BarbaraSelden/saasimages2

saas images final

Ja-Dunn committed:
Merge pull request #37819 from barbkess/topic-updates4

remove 2 migration topics

GitHubber17 committed:
Merge pull request #36439 from SnehaGunda/ASAOverview

Adding ASA overview topic

rolyon committed:
Tweaks to text, code syntax, deleted screenshot

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-psroledefobject-pr6588

v-alje committed:
Merge pull request #37829 from MicrosoftDocs/FromPublicRepo

From public repo

tfitzmac committed:
added clarification about lock

PRMerger17 committed:
Merge pull request #37820 from rachel-msft/removeext

remove extension section

kobulloc-MSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#7172

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#7172
Azure App Service link pointed to a dead link (https://plugins.jenkins.io/azure-app-server), when it looks like https://plugins.jenkins.io/azure-app-service was intended.

Commit:
Merge pull request #37831 from kobulloc-MSFT/patch-1

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#7172

Commit:
Merge pull request #37830 from tfitzmac/0413manage

added clarification about lock

Ja-Dunn committed:
Merge pull request #37813 from rolyon/rolyon-psroledefobject-pr6588

[Azure RBAC] Incorporated feedback from CarnegieJ in PR #6588

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
